### ISSUE TITLE
tcpbridge reconnect doesn't work

### ISSUE THREAD

**Issue by plajjan** [plajjan/vrnetlab#50]
```
Traceback (most recent call last):
  File "/tcpbridge.py", line 65, in work
    buf = i.recv(2048)
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tcpbridge.py", line 97, in <module>
    tt.work()
  File "/tcpbridge.py", line 67, in work
    i.connect()
TypeError: connect() takes exactly one argument (0 given)
```
think I wrote the reconnect code blind so no wonder I got it wrong :/

**Comment by plajjan** [comment]
Also, should handle late connect, i.e. we try to connect on start but the VR hasn't been brought up yet so we get connection failure:
```
kll@docker-ci:~/vrnetlab/tcpbridge$ docker logs drunk_hamilton
Traceback (most recent call last):
  File "/tcpbridge.py", line 96, in <module>
    tt.add_p2p(p2p)
  File "/tcpbridge.py", line 39, in add_p2p
    left.connect(src)
ConnectionRefusedError: [Errno 111] Connection refused
```

**Comment by plajjan** [comment]
Seems to be fixed through 820296e - closing.
